apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -86,7 +86,7 @@ interface CommonCalendarEvent extends Event {
 }
 
 export interface CourseEvent extends CommonCalendarEvent {
-    bldg: string; // E.g., ICS 174, which is actually building + room
+    bldg: string[]; // E.g., ICS 174, which is actually building + room
```

Comment:
Idea: replace `bldg: string[]` with `locations: Location[]` and add an
```
interface Location {
    building: string
    room: string
}
```
If you agree this is a good idea, please include the change in this PR. Otherwise lmk what problems you think this would introduce.

Reply:
re: showing correct locations for given days - I have no clue how we could verify this programmatically since it could be set arbitrarily by profs/schedules.

+1 re: introducing the location type (will commit that soon) although I think we can also add a `days: string[]` attribute (and potentially an enum too) for if the days of week end up being supplied programmatically later

Reply:
IDK if we can even scrape that information about which location is for which day. @ecxyzzy do you know?

Reply:
If there's a one location for one set of days (see screenshot), then the first set of days is suppposed to correspond to the first location, the second set to the second location, etc. If not, then it's arbitrary.
![image](https://github.com/icssc/AntAlmanac/assets/89349085/30680a9a-49e0-4081-aca1-ffc82c97d889)

Reply:
just curious, what class is this for testing purposes 🤔 i probably should have asked but i assumed (incorrectly) classes were searchable by only building/room

Reply:
The screenshot above corresponds to section code `44100` for 2022 Spring. Another test case that might be useful is `17085` for the same term, since it has two buildings in one meeting object.